docs/animations/ParallaxService.md

```diff
@@ -50,7 +50,7 @@ However, the way the two are used is different. Unlike the ParallaxService, the
 
 Here is an example of using the ParallaxView
 
-```csharp
+```xaml
```

Comment:
I will fix this in my PR. So you can revert it.

Reply:
There isn't any need.

Reply:
My point is you can revert all the doc changes.